src/Object/Post.php

```diff
@@ -585,17 +585,17 @@ public function getTemplateData(array $conv_responses, string $formSecurityToken
 			}
 		}
 
+		// Copy values/set defaults
 		$result['total_comments_num'] = $this->isToplevel() ? $total_children : 0;
-
 		$result['private'] = $item['private'];
 		$result['toplevel'] = ($this->isToplevel() ? 'toplevel_item' : '');
+		$result['flatten'] = false;
+		$result['threaded'] = false;
```

Comment:
Please remove the condition below.
```suggestion
		$result['flatten'] = !$this->isThreaded();
		$result['threaded'] = $this->isThreaded();
```